environments/sbox/sbox.tfvars

```diff
@@ -0,0 +1,7 @@
+env = "sbox"
```

Comment:
this should be set on the command line, in the terraform template

Reply:
resolved